features/step_definitions/purchase_steps.rb

```diff
@@ -22,9 +22,9 @@
 
 Then(/^the book is added to my cart$/) do
   @cart = @user.cart
-  expect(@cart.order_items).to_not eq(nil)
-  last_order_item = @cart.order_items.last
-  expect(last_order_item.book).to eq(@book)
+  expect(@cart.line_items).to_not eq(nil)
+  last_line_item = @cart.line_items.last
+  expect(last_line_item.book).to eq(@book)
```

Comment:
I thought the same, but all of these tests are written around predefined feature files that are part of the Bookstore assignment. I'm fairly certain I'm not supposed to modify the feature files, so I'm just building the steps to go along with them.